framework/src/controller/schema/application_config_schema.js

```diff
@@ -17,7 +17,7 @@
 module.exports = {
 	id: '#/app/config',
 	type: 'object',
-	required: ['app', 'components', 'modules'],
+	required: ['app', 'network', 'components', 'modules'],
```

Comment:
I would suggest to move `network` under `app`. It will match the code structure. Top level configurations should only stay with `app`, `components` and `modules`